lib/modules/newCommentCount.js

```diff
@@ -40,6 +40,19 @@ module.options = {
 		value: true,
 		description: 'Show the Subscribe button?',
 	},
+	monitorPostsVisited: {
+		type: 'boolean',
+		value: true,
+		description: 'Monitor the number of comments on posts you have visited.',
+		advanced: true,
+	},
+	monitorPostsVisitedIncognito: {
+		dependsOn: 'monitorPostsVisited',
+		type: 'boolean',
+		value: false,
```

Comment:
Let's make this `process.env.BUILD_TARGET !== 'firefox'` or more conservatively `process.env.BUILD_TARGET === 'chrome'` since Chrome users will have specifically enabled the extension in incognito mode and seen the persistent warning:
![image](https://cloud.githubusercontent.com/assets/7673145/18460441/7f681846-793f-11e6-9bdf-46807c2750e0.png)